src/components/implementation/tests/unit_defcompinfo/unit_defcompinfo.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2016, Phani Gadepalli and Gabriel Parmer, GWU, gparmer@gwu.edu.
+ *
+ * This uses a two clause BSD License.
```

Comment:
It looks like none of the unit tests have a license header in them. I thought that was missing, should have been there for them too. So started the trend with this. 
I also don't have a good idea for what's the right license here.